manifests/kustomize/base/pipeline/ml-pipeline-apiserver-deployment.yaml

```diff
@@ -173,4 +177,12 @@ spec:
           requests:
             cpu: 250m
             memory: 500Mi
+        volumeMounts:
+        - name: webhook-certs
+          mountPath: /etc/webhook/certs
```

Comment:
I'm not seeing where the API server uses the certificate.

Reply:
If the whole REST API is protected by TLS by this certificate, we'll need to ensure the frontend can still connect over the `ml-pipeline` `Service` as it might expect HTTP and not trust the Cert Manager CA.